.idea/misc.xml

```diff
@@ -12,5 +12,8 @@
     </writeAnnotations>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="FrameworkDetectionExcludesConfiguration">
```

Comment:
It is really needs?

Reply:
This file is not on gitignore and it's been updated in the repo, so I really didn't bother removing it.